portal/gatsby-config.js

```diff
@@ -1,5 +1,15 @@
 /* eslint-disable */
 module.exports = {
+    flags: { 
+        PRESERVE_WEBPACK_CACHE: true,
+        FAST_DEV: true,
+        DEV_SSR: true,
+        QUERY_ON_DEMAND: true,
+        LAZY_IMAGES: true,
+        PRESERVE_FILE_DOWNLOAD_CACHE: true,
```

Comment:
to keep the downloaded files cache when changing gatsby-node.js & gatsby-config.js files because, again, you rarely need to re-download them